modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationTemplateLocalServiceTest.java

```diff
@@ -109,6 +118,55 @@ public void testAddNotificationTemplate() throws Exception {
 			notificationTemplate);
 	}
 
+	@Test
+	public void testAddNotificationTemplateError() throws Exception {
+		User user = TestPropsValues.getUser();
+
+		AssertUtils.assertFailure(
+			NotificationRecipientSettingNameException.class,
+			"Notification recipient setting name is invalid",
+			() -> _notificationTemplateLocalService.addNotificationTemplate(
+				NotificationTemplateUtil.createNotificationContext(
```

Comment:
The `NotificationTemplateUtil.createNotificationContext(` is creating wrong NotificationRecipientSettingNames, could you please check if we can remove this method?